src/index.js

```diff
@@ -1,5 +1,7 @@
 import OIOProvider from './OIOProvider'
```

Comment:
Wondering  if this  style might be less verbose especially as more components are added (shouldn't require any additional babel plugins as it's vanilla ES6):

```js
export { default as OIOProvider } from './OIOProvider'
export { default as Text } from './Text'
```